book/6-export.md

```diff
@@ -0,0 +1,118 @@
+zarr_small = np_array
+
+# Do some processing with another tool / software here e.g. Fiji
+sub_array = zarr.zeros_like(zarr_small)
+
+# Then load the result:
+# sub_array = iio.imread("image_file.tiff", plugin='tifffile')
+
+original_array[40:50, 40:50, 40:50]=zarr_small # Insert non-processed data
+
+changed_array[slice_range, slice_range, slice_range] = sub_array  # Insert processed data
+
+
+```
+
+To compare the original and processed sub-volumes being inserted back into the image, we can visualize them side by side using `matplotlib`.
+
+```{code-cell} ipython3
+from data_helpers import plot_slice
+import matplotlib.pyplot as plt
+
+plot_slice(original_array, z_idx=45)
+plot_slice(changed_array, z_idx=45)
+```
+
++++
+
+## Conversion Format Comparison
+
+The following table summarizes the different file formats for converting Zarr sub-volumes, their associated tools, and trade-offs:
+
+| Format | Tool/Library | Advantages | Disadvantages | Open Source Viewers |
+|--------|--------------|------------|---------------|---------------------|
+| **TIFF** | [imageio](https://imageio.readthedocs.io/) | • Widely supported<br>• Preserves bit depth<br>• Good for microscopy workflows<br>• Fast read/write | • Limited metadata support<br>• Large file sizes<br>• No compression by default | Napari, ImageJ, Fiji |
+| **NIfTI** | [nibabel](https://nipy.org/nibabel/) | • Medical imaging standard<br>• Rich metadata (orientation, spacing)<br>• Good compression<br>• ITK-SNAP, FSL, AFNI support | • Medical imaging specific<br>• Limited to certain data types<br>• Complex coordinate systems | ITK-SNAP, FSL, AFNI |
+| **DICOM** | [pydicom](https://pydicom.github.io/) | • Clinical standard<br>• Extensive metadata<br>• Patient/study information<br>• Universal medical viewer support | • Complex format<br>• Large overhead<br>• Requires many mandatory fields<br>• Slow for large volumes | OsiriX, RadiAnt, Horos |
+| **NumPy** | [numpy](https://numpy.org/) | • Direct array format<br>• Fast loading<br>• Preserves exact data<br>• Python native | • No metadata<br>• Large file sizes<br>• Python ecosystem only | None (Python only) |
+
+### Recommended Use Cases
+
+- **TIFF**: General imaging workflows, microscopy data, when broad tool compatibility is needed
+- **NIfTI**: Medical imaging analysis, neuroimaging studies, when spatial metadata is critical
+- **DICOM**: Clinical workflows, when patient metadata is required, for regulatory compliance
+- **NumPy**: Quick prototyping, Python-only workflows, temporary data exchange
+
+
+
+Both the NIfTI and DICOM sub-volumes can be viewed in open-source viewers such as `ITK-SNAP` (see [ITK-SNAP site](https://www.itksnap.org/pmwiki/pmwiki.php)).
+
+Note: This process is more complex for OME-Zarr images that have a series of different resolution levels.
+When writing the processed sub-volume back to the OME-Zarr image, you will need to ensure that the resolution levels are updated correctly.
```

Comment:
this warning should go right up near the top of the chapter, so readers understand up front this is only applicable to Zarr